### PR TITLE
rules for OCP-25793

### PR DESCRIPTION
**Pull request by SSshahan**
@yapei  please help to review, thanks!

### CODE REVIEW

**Comment by SSshahan** [comment]
@yapei  please review, thanks!

**Comment by pruan-rht** [comment]
@SSshahan there are conflicts

File: lib/rules/web/admin_console/4.3/common_ui_elements.xyaml [hunk 1]
```diff
@@ -121,6 +121,11 @@ click_kebab_of_one_resource:
 click_one_operation_in_kebab:
   action: click_kebab_of_one_resource
   action: click_button
+check_field_on_resource_list_page:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: I would call this `check_column_in_table`

File: lib/rules/web/admin_console/4.3/common_ui_elements.xyaml [hunk 2]
```diff
@@ -392,10 +398,13 @@ check_resource_details:
...
     if_param: current_configuration
     ref: check_current_configuration
+  action:
+    if_param: created_at
+    ref: check_created_at
 check_resource_details_key_and_value:
   element: &key_value
     selector:
-      xpath: //dt[contains(text(),'<key>')]/following-sibling::dd[1][contains(.,'<value>')] | //button[(text()='<key>')]/parent::dt/following-sibling::dd[1][contains(.,'<value>')]
+      xpath: //dt[contains(text(),'<key>')]/following-sibling::dd[1][contains(.,'<value>')] | //button[(text()='<key>')]/parent::dt/following-sibling::dd[1][contains(.,'<value>')] | //button[(text()='<key>')]
```
Comment (yapei) [inline lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: I would say not update this
```
$x("//dt[contains(text(),'Update Strategy')]/following-sibling::dd[1][contains(.,'Custom')] | //button[(text()='Update Strategy')]/parent::dt/following-sibling::dd[1][contains(.,'Custom')] | //button[(text()='Update Strategy')]")
```
Array [ button.pf-c-button.pf-m-plain.co-m-pane__details-popover-button ]

File: lib/rules/web/admin_console/4.3/common_ui_elements.xyaml [hunk 3]
```diff
@@ -121,6 +121,11 @@ click_kebab_of_one_resource:
 click_one_operation_in_kebab:
   action: click_kebab_of_one_resource
   action: click_button
+check_field_on_resource_list_page:
+  elements:
+  - selector:
+      xpath: //th[contains(@data-label,'<field>')]
+    timeout: 30
```
Comment (yapei) [inline lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: Are we waiting table loaded completely here? If yes, I would say removing timeout from this action and use another step `wait_table_loaded` in feature, that looks much meaningful

File: lib/rules/web/admin_console/4.3/common_ui_elements.xyaml [hunk 4]
```diff
@@ -245,6 +250,7 @@ check_link_and_text:
   element:
     selector:
       xpath: //a[contains(.,'<text>') and contains(@href,'<link_url>')]
+    timeout: 30
```
Comment (yapei) [inline lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: Same as above

**Comment by SSshahan** [comment]
@yapei  Updated, please review again.   http://pastebin.test.redhat.com/825099 Will give Jenkins log later.

**Comment by yapei** [comment]
Please resolve conflicts

**Pull request merged**